content/en/contribute/code/core/automated-tests.md

```diff
@@ -201,7 +201,8 @@ To run the upgrade e2e tests in your local environment, follow these steps:
     - `export MARKET_URL_READ=https://staging.dev.medicmobile.org`.
     - `export STAGING_SERVER=_couch/builds_4`.
     - `export BRANCH=<your branch name>`.
-    - `export BASE_VERSION=<base version>`(it can be used `latest` as the value).
+    - `export BASE_VERSION=<CHT base version>`(it can be used `latest` as the value).
+    - `export TAG=<CHT version>`(e.g. `4.8.1`).
```

Comment:
This makes it look like the TAG is required, but it's not.

Reply:
I've added `Optional`

Reply:
@dianabarsan can I merge?